src/main/scala/com/advancedtelematic/campaigner/data/DataType.scala

```diff
@@ -176,9 +176,9 @@ object DataType {
    *   to `processed` state in the UI until a device goes to the Director)
    * - `rejected` when a device is rejected by Director and is not a part of the
    *    campaign anymore (corresponds to `not impacted` state in the UI)
-   * - `scheduled` when a device is approved by Director and is scheduled for
-   *    an update (partly corresponds to `queued` state in the UI)
-   * - `accepted` when an update was accepted on a device and is about to be
+   * - `scheduled` when a device is scheduled for an approval by user and
+   *    Director (partly corresponds to `queued` state in the UI)
```

Comment:
Should just be `device has been assigned an update by Director`. There's no user input at this state.

Reply:
What you've described actually corresponds to `accepted` state. `scheduled` is the state we're in after sending an assignment request to director, but before receiving approval/rejection response from it.